weaviate/collections/classes/config.py

```diff
@@ -916,6 +921,9 @@ def aws(
         endpoint: Optional[str] = None,
         service: Union[AWSService, str] = "bedrock",
         max_tokens: Optional[int] = None,
+        target_model: Optional[str] = None,
+        target_variant: Optional[str] = None,
+        temperature: Optional[float] = None,
```

Comment:
if they are sagemaker only, could we add them to a aws_sagemaker factory? Ideally together with aws_bedrock and deprecate this one